tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -23,6 +23,82 @@
 from robottelo.constants import FOREMAN_PROVIDERS
 
 
+_provider_parameteres = [
+    {
+        'provider': 'vmware',
+        'params': {
+            'provider': FOREMAN_PROVIDERS['vmware'],
+            'provider_content.vcenter': settings.vmware.vcenter,
```

Comment:
The issue about reading config in import time is when generating docs since usually the config file is not present on git, just an example or template file. That would raise an exception for not being able to find the file. If you are not generating docs than probably you won't face this.

Usually making reading files and other related stuff lazy is better since it would make importing modules quick and avoid doing extra work that could be delayed.